src/conversion_helper.js

```diff
@@ -62,6 +62,7 @@ const metricDistanceUnits = [/\bkm\b/, /light-?years?/,
                              /(?:milli|centi|deca|kilo)?met(?:re|er)s?/];
 const metricWeightUnits = [/kgs?/, /grams?/, /kilograms?/];
 const metricVolumeUnits = [/(?:milli|centi|deca|kilo)?lit(?:er|re)s?/, /(?:deca|kilo)?m\^3/];
+const metricForceUnits = [/newtons?/, /N/, /dynes?/];
```

Comment:
Oops, sorry I just noticed this now, can you remove the /n/ regex? Any subreddit, post title, or comment with an "n" in it will be ignored, which is not ideal. This brings up a good point that the code here is poorly documented, I'll work on improving its clarity so this doesn't happen in the future :) Thank you!!